specification/resources/gen-ai/definitions.yml

```
@@ -0,0 +1,2835 @@
+  type: object
+apiAgentTemplate:
+  description: Represents an AgentTemplate entity
+  properties:
+    created_at:
+      description: The agent template's creation date
+      example: "2023-01-01T00:00:00Z"
+      format: date-time
+      type: string
+    description:
+      description: Description of the agent template
+      example: example string
+      type: string
+    instruction:
+      description: Instructions for the agent template
+      example: example string
+      type: string
+    k:
+      description: The 'k' value for the agent template
+      format: int64
+      type: integer
+    knowledge_bases:
+      description: List of knowledge bases associated with the agent template
+      items:
+        $ref: '#/apiKnowledgeBase'
+      type: array
+    max_tokens:
+      description: The max_tokens setting for the agent template
+      format: int64
+      type: integer
+    model:
+      $ref: '#/apiModel'
+      description: The model associated with the agent template
+    name:
+      description: Name of the agent template
+      example: example name
+      type: string
+    temperature:
+      description: The temperature setting for the agent template
+      example: 123
+      format: float
+      type: number
+    top_p:
+      description: The top_p setting for the agent template
+      example: 123
+      format: float
+      type: number
+    updated_at:
+      description: The agent template's last updated date
+      example: "2023-01-01T00:00:00Z"
+      format: date-time
+      type: string
+    uuid:
+      description: Unique id
+      example: 123e4567-e89b-12d3-a456-426614174000
+      type: string
+  type: object
+apiAgreement:
+  description: Agreement Description
+  properties:
+    description:
+      example: example string
+      type: string
+    name:
+      example: example name
+      type: string
+    url:
+      example: example string
+      type: string
+    uuid:
+      example: 123e4567-e89b-12d3-a456-426614174000
+      type: string
+  type: object
+apiAuditHeader:
+  description: An alternative way to provide auth information. for internal use only.
+  properties:
+    actor_id:
+      example: 12345
+      format: uint64
+      type: string
+    actor_ip:
+      example: example string
+      type: string
+    actor_uuid:
+      example: 123e4567-e89b-12d3-a456-426614174000
+      type: string
+    context_urn:
+      example: example string
+      type: string
+    origin_application:
+      example: example string
+      type: string
+    user_id:
+      example: 12345
+      format: uint64
+      type: string
+    user_uuid:
+      example: 123e4567-e89b-12d3-a456-426614174000
+      type: string
+  type: object
+apiBatchJobPhase:
+  default: BATCH_JOB_PHASE_UNKNOWN
+  enum:
+  - BATCH_JOB_PHASE_UNKNOWN
+  - BATCH_JOB_PHASE_PENDING
+  - BATCH_JOB_PHASE_RUNNING
+  - BATCH_JOB_PHASE_SUCCEEDED
+  - BATCH_JOB_PHASE_FAILED
+  - BATCH_JOB_PHASE_ERROR
+  example: example string
```

```suggestion
  example: BATCH_JOB_PHASE_PENDING
```

A number of the validation errors I'm seeing are for cases like this where the example does not match a valid enum value.